apps/src/sites/code.org/pages/public/teacher-dashboard/index.js

```diff
@@ -221,6 +221,7 @@ function main() {
 
```

Comment:
I haven't changed this route.  I think it works because the base URL doesn't change during navigation, only the URL hash.